src/backend/distributed/sql/downgrades/citus--12.0-1--11.3-1.sql

```diff
@@ -41,3 +41,6 @@ DROP FUNCTION pg_catalog.citus_internal_unregister_tenant_schema_globally(Oid, t
 #include "../udfs/citus_drop_trigger/10.2-1.sql"
 
 DROP TABLE pg_catalog.pg_dist_tenant_schema;
+
+#include "../udfs/citus_tables/11.1-1.sql"
+#include "../udfs/citus_shards/11.1-1.sql"
```

Comment:
Need to have those two lines before `DROP TABLE pg_catalog.pg_dist_tenant_schema;`, this should fix the regression tests.